docs/tutorial.rst

```diff
@@ -242,5 +242,151 @@ effective population size from the **TODO add crossref to catalog section**
 .. :ref:`sec_catalog_homo_sapiens_genome`
 
 
+************************************************
+Analysis: Genetic divergence between populations
+************************************************
 
+In this tutorial, we will simulate some samples of human chromosomes
+from different populations,
+and then estimate the genetic divergence between each population pair. 
 
+First, let's use the ``--help-models`` option to see the selection of demographic
+models available to us:
+
+.. code-block:: console
+
+    $ stdpopsim homsap --help-models
+
+This prints detailed information about all of the available models to
+the terminal.
+In this tutorial, we will use the model of African-American admixture from
+`2011 Browning et al <http://dx.doi.org/10.1371/journal.pgen.1007385>`_.  
+From the help output (or the :ref:`Catalog <sec_catalog>`),
+we can see that this model has id ``america``,
```

Comment:
Is there a way to automatically get this id? I only say this because there was talk of changing the id's. So, after the id's are settled on, we'll have to change it here.

Reply:
Not that I know of. Btw, I'm not a bit fan of this model name for other reasons (#235) so would be in favour of some change -- I'll just try to keep an eye out and update the docs if this happens